src/files/BrsFile.spec.ts

```diff
@@ -2134,18 +2134,19 @@ describe('BrsFile', () => {
 
     describe('type definitions', () => {
         it('only exposes defined functions even if source has more', async () => {
-            await program.addOrReplaceFile('source/main.d.bs', `
```

Comment:
Swapped the order of the file loading, so we can ensure the file was fully parsed (since it didn't know about the .d.bs file)